### PR TITLE
chore: do not limit acceptance test pods

### PR DESCRIPTION
**Pull request by rokroskar**
Limits on acceptance test pods seem to introduce throttling and/or container restarts. Remove the limits to see if this might resolve some of the test failures. 

This PR is testing the resource request value changes proposed here: https://github.com/SwissDataScienceCenter/ops-dev.renku.ch/pull/194

/deploy

### CODE REVIEW

**Comment by rokroskar** [comment]
Actually it doesn't seem like the test container ever punches above the CPU limit except for at initialization time: 

image


Still, I don't see much sense in not using the Burstable QOS class.

**Comment by ableuler** [comment]
oh, how did you get to 2.5 CPU? I've monitored this quite a bit and had the impression that with request/limit being 1.5 CPU it always had some margin.

File: helm-chart/renku/values.yaml [hunk 1]
```diff
@@ -654,10 +654,6 @@ tests:
       memory: "3G"
       cpu: "1500m"
```
Comment (ableuler) [inline helm-chart/renku/values.yaml:655]: why not 2500m here according to your test?

**Comment by rokroskar** [comment]
oh - that initial spike doesn't really matter so much. my take is that it's an artifact of the test starting. For the rest of the test the request could even be a bit lower. The point is more to remove the limits to not throttle the container unnecessarily.

File: helm-chart/renku/values.yaml [hunk 2]
```diff
@@ -654,10 +654,6 @@ tests:
...
       ephemeral-storage: "3G"
-    limits:
```
[the remark this answers is not in the document]
Comment (rokroskar) [reply helm-chart/renku/values.yaml:657]: yeah the problem is actually not so much the limit as the request - it's too high, which makes the reservation for the deployment too high in the end and the pod more difficult to schedule

[the remark this answers is not in the document]
Comment (ableuler) [reply helm-chart/renku/values.yaml:657]: If you're convinced that throttling the pod during that initial spike won't lead to failures I'm all for reducing the request again.

[the remark this answers is not in the document]
Comment (rokroskar) [reply helm-chart/renku/values.yaml:657]: I'm convinced I'm fed up with over-provisioning resources... 😆

[the remark this answers is not in the document]
Comment (rokroskar) [reply helm-chart/renku/values.yaml:657]: this is a random sampling of acceptance test cpu consumption from a few days back:

image

After the initial spike they all need ~0.5 cores

[the remark this answers is not in the document]
Comment (ableuler) [reply helm-chart/renku/values.yaml:657]: > I'm convinced I'm fed up with over-provisioning resources... 😆

Random test failures aren't really cool either. But if the acceptance tests can live with 0.5 CPU what you're proposing is obviously much better than the current configuration. One thing though: if we're going for 0.5 CPU, I _would_ actually put the limit back and set it to 0.5 CPU as well. This makes sure that we're always throttling the tests to 0.5 CPU and not just when the test happens to be scheduled on a full node.

[the remark this answers is not in the document]
Comment (rokroskar) [reply helm-chart/renku/values.yaml:657]: > if we're going for 0.5 CPU, I would actually put the limit back and set it to 0.5 CPU as well. This makes sure that we're always throttling the tests to 0.5 CPU and not just when the test happens to be scheduled on a full node.

The nodes are full only on paper and always under-utilized. Therefore a spike of 1-2 cores for ~1-2 minutes isn't going to matter. That's my current take anyway. 😆

File: .github/workflows/pull-request-test.yml [hunk 3]
```diff
@@ -82,7 +82,7 @@ jobs:
           RENKU_ANONYMOUS_SESSIONS: true
           RENKU_RELEASE: ci-renku-${{ github.event.number }}
           RENKU_VALUES_FILE: "${{ github.workspace }}/values.yaml"
-          RENKU_VALUES: ${{ secrets.CI_RENKU_VALUES }}
+          RENKU_VALUES: ${{ secrets.CI_VALUES_FOR_TESTING_RESOURCE_SETTINGS }}
```
Comment (ableuler) [inline .github/workflows/pull-request-test.yml:85]: Time to undo the change in this line and merge the PR 👍

**Review by ableuler — CHANGES_REQUESTED** [review]

File: .github/workflows/pull-request-test.yml (hunk 3 above)
Comment (rokroskar) [reply .github/workflows/pull-request-test.yml]: done

**Review by ableuler — DISMISSED** [review]

File: .github/workflows/pull-request-test.yml (hunk 3 above)
Comment (rokroskar) [reply .github/workflows/pull-request-test.yml]: of course now they refuse to run 🙈

**Comment by rokroskar** [comment]
@jachro @ableuler i cherry-picked the test fix into https://github.com/SwissDataScienceCenter/renku/pull/2409

**Review by ableuler — APPROVED** [review]
